src/button-group/interfaces.ts

```diff
@@ -90,7 +92,20 @@ export interface ButtonGroupProps extends BaseComponentProps {
    * * `text` (string) - The name of the group rendered as ARIA label for this group.
    * * `items` ((ButtonGroupProps.IconButton | ButtonGroupProps.MenuDropdown)[]) - The array of items that belong to this group.
    */
-  items: ReadonlyArray<ButtonGroupProps.ItemOrGroup>;
+  items?: ReadonlyArray<ButtonGroupProps.ItemOrGroup>;
+  /**
+   * Instead of passing `items`, group items can be passed as children. When using this,
+   * you will need to manage click handlers and state of buttons.
+   *
+   * Each item should contain a single focusable element which must have a unique `data-itemid` attribute.
```

Comment:
This sentence is a bit confusing, until one sees the example. Can we say it more explicitly, e.g. "Pass interactive elements directly to the children prop as a React elements array. Every item in the array ..."?